src/images/svg/info2.svg

```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
please move this to the specific folder for mt5 to reduce svg population in folder 😅